paymentsheet/src/main/java/com/stripe/android/link/ui/wallet/WalletViewModel.kt

```diff
@@ -98,6 +100,13 @@ internal class WalletViewModel @Inject constructor(
             is LinkLaunchMode.Authentication -> null
         }
 
+    private val LinkLaunchMode.paymentSelectionHint: String?
+        get() = (this as? LinkLaunchMode.PaymentMethodSelection)?.hint
+            ?.takeIf {
+                FeatureFlags.forceEnableLinkPaymentSelectionHint.isEnabled ||
+                    configuration.enablePaymentSelectionHint
```

Comment:
if this ends up coming as a link flag, you can do `LinkConfiguration.flags[...]` and avoid having an elements session computed property etc. 

(ideally we pass the settings object to `LinkConfiguration` at some point to facilitate populating settings)